doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -82,175 +76,107 @@ To work with the nRF54H20 DK, follow the instructions in the next sections to in
-
-      1. Open Git Bash.
-      #. Download and run the :file:`bootstrap-toolchain.sh` installation script file using the following command:
-
-         .. parsed-literal::
-            :class: highlight
-
-            curl --proto '=https' --tlsv1.2 -sSf https://files.nordicsemi.com/artifactory/swtools/external/scripts/bootstrap-toolchain.sh | NCS_TOOLCHAIN_VERSION=v2.6.99-cs2 sh
-
-         Depending on your connection, this might take some time.
-      #. Open a new terminal window with the |NCS| toolchain environment by running the following command:
-
-         .. parsed-literal::
-            :class: highlight
-
-            c:/ncs-lcs/nrfutil.exe toolchain-manager launch --terminal --chdir "c:/ncs-lcs/work-dir" --ncs-version v2.6.99-cs2
-
-         This setup allows you to access west and other development tools.
-         Alternatively, you can set up the environment variables manually by running the following command::
-
-            c:/ncs-lcs/nrfutil.exe toolchain-manager env --as-script
-
-         Copy-paste the output into the terminal and execute it to enable the use of west directly in that window.
-
-         .. note::
-            WWhen working with west in the customer sampling release, you must always use a terminal window with the |NCS| toolchain environment.
-
-      If you run into errors during the installation process, delete the :file:`.west` folder inside the :file:`C:\\ncs-lcs` directory, and start over.
-
-   .. tab:: Linux
-
-      1. Open a terminal window.
-      #. Download and run the :file:`bootstrap-toolchain.sh` installation script file using the following command:
-
-         .. parsed-literal::
-            :class: highlight
-
-            curl --proto '=https' --tlsv1.2 -sSf https://files.nordicsemi.com/artifactory/swtools/external/scripts/bootstrap-toolchain.sh | NCS_TOOLCHAIN_VERSION=v2.6.99-cs2 sh
-
-         Depending on your connection, this might take some time.
-      #. Open a new terminal window with the |NCS| toolchain environment by running the following command:
-
-         .. parsed-literal::
-            :class: highlight
-
-            $HOME/ncs-lcs/nrfutil toolchain-manager launch --shell --chdir "$HOME/ncs-lcs/work-dir" --ncs-version v2.6.99-cs2
-
-         .. note::
-            When working with west in the customer sampling release, you must always use a shell window with the |NCS| toolchain environment.
+Installing the |NCS| v2.7.0 and its toolchain
+*********************************************
 
-      If you run into errors during the installation process, delete the :file:`.west` folder inside the :file:`ncs-lcs` directory, and start over.
+You can install the |NCS| v2.7.0 and its toolchain by using Toolchain Manager.
 
-   .. tab:: macOS
+Toolchain Manager is a tool available from `nRF Connect for Desktop`_, a cross-platform tool that provides different applications that simplify installing the |NCS|.
+Both the tool and the application are available for Windows, Linux, and MacOS.
```

Comment:
Last time I checked with @ihansse , Toolchain Manager is recommended only for NCS v1.9.0 and earlier, and so we state in the docs about it: https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/installation/install_ncs.rst#legacy-installation-with-toolchain-manager

Having this information here contradicts the info I got from Ingar, so I assume it is incorrect.
Also, you have all the installation instructions on the page above I linked.

Reply:
This is valid only for lilium, and supposedly just for 2.7